bucket/simple-dnscrypt.json

```diff
@@ -40,10 +52,10 @@
     "pre_uninstall": [
         "if (!(is_admin)) { error \"$app requires admin rights to $cmd\"; break }",
         "",
-        "Start-Process -Wait \"$dir\\Uninstall.exe\" | Out-Null",
+        "Start-Process -Wait \"$dir\\Uninstall.exe\" | Out-Null; Start-Sleep -Seconds 8",
```

Comment:
```suggestion
        "Start-Process -Wait \"$dir\\Uninstall.exe\" | Out-Null",
```

Reply:
@HUMORCE I can't remove this. The previous uninstallation process would cause users to lose access to the Wi-Fi/Internet permanently, unless simple-dnscrypt was reinstalled.

Reply:
I think this manifest should be moved to Nonportable bucket.

Reply:
No, I don't think it needs to be. If you kept my edit, the manifest would work fine for everybody else.

Reply:
it's better not to use `Start-Sleep`.